src/ophys_etl/modules/denoising/fine_tuning/schemas.py

```diff
@@ -0,0 +1,65 @@
+import argschema
+from deepinterpolation.cli.schemas import FineTuningInputSchema, \
+    GeneratorSchema
+
+from ophys_etl.schemas.fields import H5InputFile
+
+
+class DataSplitterInputSchema(argschema.ArgSchema):
+    movie_path = H5InputFile(
+        required=True,
+        description='Path to ophys movie'
+    )
+    train_frac = argschema.fields.Float(
+        default=0.7,
+        description='Amount of data to set aside for training. The rest will '
+                    'be used for validation'
+    )
+    seed = argschema.fields.Int(
+        default=None,
```

Comment:
Do you want to default to a seed or set that at runtime when we are processing data?

Reply:
I think it can be passed unless you prefer it to be set here so we don't forget.

Reply:
I'd say set a default. Reproducibility is great to have even if you are just casually testing it.